### DISCUSSION TITLE
just wanted to say thanks!

### DISCUSSION THREAD

**Discussion by atomGit** [nicotine-plus/nicotine-plus#2621] [General]
i'm kind of new to the soulseek network and nicotine+ seems like a very nice client and an ambitious project

often the only feedback a dev gets is negative/bugs, so i just wanted to thank the dev(s) and all the contributors :)

**Comment by mathiascode** [comment]
Thank you! I've lost count of how many thousands of hours I've put into this project, but reading messages like this makes it worthwhile. :)

**Comment by atomGit** [comment]
you're most welcome! i'm enjoying both the network and the client - the social aspect of the network is interesting - i think that adds a lot to its popularity

i also love how the name originated :)

i'm a sort of "developer" myself, though only bash projects, and i can hardly imagine how complex a project like yours is - thanks for keeping it going!

there's an option to set the server (server.slsknet.org) and so i briefly poked around for alt servers but didn't find any - are there any that are worthwhile using?

**Reply by mathiascode** [reply]
> i can hardly imagine how complex a project like yours is - thanks for keeping it going!

Nicotine+ (and Soulseek clients in general) is definitely interesting to work on, considering its age (22 years with many previous developers!) and all the features you have to deal with, such as networking (P2P), file transfers, file sharing, search, chat... You definitely learn a lot!

A lot of time has been spent modernizing the client and making the codebase more modular and maintainable, so we're in a pretty good position.

 > there's an option to set the server (server.slsknet.org) and so i briefly poked around for alt servers but didn't find any - are there any that are worthwhile using?

There aren't any third-party servers at the moment. The option mainly exists for development purposes, as well as an escape hatch in case something ever happens to the official server. https://github.com/seeschloss/soulfind mostly works, but hasn't seen any real development since 2006 or so, and isn't very reliable with many users.
